src/transform/Polyfill.js

```diff
@@ -17,6 +17,14 @@ const matchesSelector = (el, selector) => {
   return false
 }
 
+/**
+ * @param {!Element} element
+ * @param {!string} selector
+ * @return {!Element[]}
+ */
+const querySelectorAll = (element, selector) =>
```

Comment:
Thoughts on maybe extending prototype on HTMLElement with something like `wmf_querySelectorAll` so don't have to pass `element`? Was wondering about this earlier...